### PR TITLE
Book class

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
Use JavaScript class approach to add, display and remove books from local storage. Built with HTML and JAVASCRIPT.

### CODE REVIEW

**Comment by Ugboaja-Uchechi** [comment]
Made sure there are no linter errors

File: main.js [hunk 1]
```diff
@@ -1,57 +1,67 @@
...
+   };
+
+   clear = function () {
+     document.getElementById(selectors.title).value = '';
+     document.getElementById(selectors.author).value = '';
+     document.getElementById(selectors.isbn).value = '';
+   };
+
+   showAlert = function (message, className) {
+     // create a div
```
Comment (rloterh) [inline main.js:43]: - [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

File: main.js [hunk 2]
```diff
@@ -1,57 +1,67 @@
...
+     list.innerHTML = `
+    <td>${book.title}</td>
+    <td>${book.author}</td>
+    <td>${book.isbn}</td>
+    <td><button class="delete">Remove</button></td>
+  `;
+     document.getElementById('book-list').appendChild(list);
+   };
+
+   clear = function () {
```
Comment (rloterh) [inline main.js:36]: - [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

File: main.js [hunk 3]
```diff
@@ -1,57 +1,67 @@
...
+     const container = document.querySelector('.container');
+     const form = document.querySelector('#book-form');
+     container.insertBefore(div, form);
+     // set the timeout
+     setTimeout(() => {
+       div.remove();
+     }, 1000);
+   };
+
+   delete = function (target) {
```
Comment (rloterh) [inline main.js:58]: - [x] Two elements are **not able to be removed** from the local storage when the _remove button_ is pressed and the page is refreshed. Kindly ensure to improve upon this aspect

**Undeletable entries:**
image

File: main.js [hunk 4]
```diff
@@ -1,57 +1,67 @@
 /* eslint-disable linebreak-style */
```
Comment (rloterh) [inline main.js:1]: - [ ] OPTIONAL
Project details on your _ReadMe_ do not match the current project. Kindly endeavor to update your ReadMe to correspond with the current project so as to make your _ReadMe_ professional.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Victor, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Correctly refactored methods and properties into classes
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @Ugboaja-Uchechi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**